src/dev/build/tasks/os_packages/package_scripts/post_install.sh

```diff
@@ -35,7 +38,7 @@ case $1 in
       IS_UPGRADE=true
     fi
 
-    set_access
+    setup
```

Comment:
Why do we no longer want to call this for Red Hat?

Reply:
fixed with a2ba7f4.  I haven't been able to get a build through pending the x-pack master failure, so not quite ready for testing yet.  Will update.

Reply:
tracking ^ at https://github.com/elastic/kibana/pull/81641